test/run_tests.sh

```
@@ -20,5 +20,22 @@ else
-exit $?
\ No newline at end of file
+# Capture pytest exit code
+PYTEST_EXIT_CODE=$?
+
+echo ""
+echo "Running C unit tests via Makefile..."
+echo "=================================="
+cd ..
+make test
+MAKE_EXIT_CODE=$?
+cd test
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)